sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -37,57 +36,56 @@ export interface TryAddOptions {
 }
 
 /**
- * A interface representing a batch of events which can be passed to the `send` method of a `EventProducer` instance.
- * This batch is ensured to be under the maximum message size supported by Azure Event Hubs service.
- *
- * Use `createBatch()` method on the `EventHubProducer` to create an instance of `EventDataBatch`
- * instead of using `new EventDataBatch()`. You can specify an upper limit for the size of the batch
- * via options when calling `createBatch()`.
- *
- * Use the `tryAdd` function on the EventDataBatch to add events to the batch. This method will return
- * `false` after the upper limit is reached, therefore check the result before calling `tryAdd()` again.
+ * A interface representing a batch of events which can be used to send events to Event Hub.
+ * 
+ * To create the batch, use the `createBatch()` method on the `EventHubProducerClient`.
+ * To send the batch, use the `sendBatch()` method on the same client.
+ * To fill the batch, use the `tryAdd()` method on the batch itself.
+ * 
  */
 export interface EventDataBatch {
   /**
-   * The partitionKey set during `EventDataBatch` creation. This value is hashed to
-   * produce a partition assignment when the producer is created without a `partitionId`
-   * If this value is set then partitionId can not be set.
+   * A value that is hashed and used by the Azure Event Hubs service to determine the partition to
+   * which the events are sent. Use the `createBatch()` method on the `EventHubProducerClient` to
+   * set the partitionKey.
    * @readonly
    * @internal
    * @ignore
    */
   readonly partitionKey?: string;
 
   /**
-   * The partitionId set during `EventDataBatch` creation.
-   * If this value is set then partitionKey can not be set.
+   * Id of the partition to which the batch of events are sent. Use the `createBatch()` method on
+   * the `EventHubProducerClient` to set the partitionId.
    * @readonly
    * @internal
    * @ignore
    */
   readonly partitionId?: string;
 
   /**
-   * Size of the `EventDataBatch` instance after the events added to it have been
-   * encoded into a single AMQP message.
+   * Size of the batch in bytes after the events added to it have been encoded into a single AMQP
+   * message.
    * @readonly
    */
   readonly sizeInBytes: number;
 
   /**
-   * Number of events in the `EventDataBatch` instance.
+   * Number of events added to the batch.
    * @readonly
    */
   readonly count: number;
 
   /**
-   * The maximum size of the batch, in bytes.
+   * The maximum size of the batch, in bytes. The `tryAdd` function on the batch will return `false`
+   * after the size of the batch reaches this limit. Use the `createBatch()` method on
```

Comment:
```suggestion
   * if the event being added causes the size of the batch to exceed this limit. Use the `createBatch()` method on
```

Making this suggestion since it's possible to have a large event that can't be added, but other events that can still be added.